src/System.Collections/src/System/Collections/Generic/SortedList.cs

```diff
@@ -453,7 +453,7 @@ public bool ContainsValue(TValue value)
 
             if (arrayIndex < 0 || arrayIndex > array.Length)
             {
-                throw new ArgumentOutOfRangeException("arrayIndex", SR.ArgumentOutOfRange_NeedNonNegNum);
+                throw new ArgumentOutOfRangeException("index", arrayIndex, SR.ArgumentOutOfRange_Index);
```

Comment:
That's the suggested name for the ICollection interface. I undid this change and pushed another commit :)